plugins/bigquery-api-backend/src/service/router.ts

```diff
@@ -38,6 +38,7 @@ export async function createRouter(
   const projectId = readAppConfig.getString('projectId');
   const gbdataset = readAppConfig.getString('gbdataSet');
   const { GoogleAuth, Impersonated } = require('google-auth-library');
+  const gbfolderId = readAppConfig.getString('gbfolderId');
```

Comment:
We want to be able to get the folderId from the entity in Backstage. This should not come from the app-config but is fine for right now.

Reply:
Okay, its hardly matter from where we are supplying params.
we can do it later.